modules/cardano-edgenode.nix

```diff
@@ -0,0 +1,52 @@
+{ accessKeyId, region, systemStart, globals }:
+
+with (import ./../lib.nix);
+
+let
+  instancesPerNode = 5;
+  params = {
+    name = "wallet";
+  };
+  mkNode = publicIP: nodes: index: {
+    name = "instance${toString index}";
+    value = {
+      autoStart = true;
+      privateNetwork = false;
```

Comment:
and just delete this file, its been replaced by `cardano-benchmark.nix`

Reply:
Done.